javascript/javascript1/week1/smart-ease.js

```diff
@@ -0,0 +1,87 @@
+//FreeCOdeCamp Exercise link
+//https://www.freecodecamp.org/myselfankitaa
+
+//Age-ify (A future age calculator)
+const yearOfBirth = 1988;
+let yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2020;
+let dogYearFuture = 2045;
+const dogAge = dogYearFuture - dogYearOfBirth;
+let dogYear = dogAge * 7;
+let shouldShowResultInDogYears = true;
```

Comment:
Be conscious of using `let` and `const` we naming your variables. If the variable is going to be constant and not be re-assigned in the future then use `const`.

If you want to let a variable be re-assigned in the future then use `let`.

Reply:
Ok, I understood and made the changes accordingly.